query-engine/core-tests/tests/query_validation_tests.rs

```diff
@@ -19,8 +19,7 @@ fn run_query_validation_test(query_file_path: &str) {
         .chain(ALL_PREVIEW_FEATURES.hidden_features())
         .collect();
     let parsed_schema = psl::parse_schema(schema).unwrap();
-    let prisma_models_schema = prisma_models::convert(Arc::new(parsed_schema));
```

Comment:
Do we still need `prisma_models::convert` at all? I don't see it being removed

Reply:
We still need it in `schema::build()`, that's the only place. It does basically nothing, but we depend on the `InternalDataModel` type for a bunch of things prisma-models does.